prep-javascript-dom-manipulation/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript DOM Manipulation</title>
+  <style>
+    #container {
+      border: 2px solid green;
+    }
+  </style>
+</head>
+<body>
+  <div id="container">
+    <p class="first-paragraph">
+      I really like coding.
+    </p>
+    <p class="secondParagraph"> I also really like to watch television and eating pancakes.
+    </p>
+    <p class="thirdParagraph"> I hope one day to meet people that also like coding, television, and pancakes.
+    </p>
```

Comment:
For this exercise we want to use the JavaScript code to assign the new `.textContent` instead of editing the HTML directly.

```js
$secondParagraph.textContent = 'I also really like to watch television and eating pancakes.';
```

Reply:
<img width="1440" alt="Captura de Pantalla 2022-10-04 a la(s) 13 05 36" src="https://user-images.githubusercontent.com/113643160/193915631-11dfcc60-1f89-47f9-a5eb-ab3cb028e0ba.png">